src/utils/viewer/viewer.cpp

```diff
@@ -388,12 +394,15 @@ Viewer::Viewer(const Arguments& arguments)
     debugBullet_ = true;
   }
 
+  gfxReplayRecordFilepath_ = args.value("gfx-replay-record-filepath");
```

Comment:
Could use a path join to support relative and global paths. Maybe this is already in the save function?

Reply:
I pass the filepath string directly to io::writeJsonToFile. With the code as is, you can do relative or absolute path like so:

./build/utils/viewer/viewer /my_stage.glb --gfx-replay-record-filepath ./relative/path/to/my_replay.json
./build/utils/viewer/viewer /my_stage.glb --gfx-replay-record-filepath /absolute/path/to/my_replay.json

The relative path is relative to the current working directory where you're running viewer.

Did you have something else in mind?

Reply:
That's fine. Just checking since we do this join explicitly for some other arguments (e.g. `"physics-config"` and `"object-dir"`).